Dockerfile

```diff
@@ -31,7 +31,7 @@ COPY glam /app/glam
 
 
 # FRONTEND BUILDER IMAGE
-FROM node:lts-slim AS frontend
+FROM node:12 AS frontend
```

Comment:
I would just be careful here since nodejs 12 has an end of life in Apr 2022:
https://nodejs.org/en/about/releases/

But otherwise r+